src/chigame/users/tables.py

```diff
@@ -0,0 +1,12 @@
+import django_tables2 as tables
+from .models import User
+
+class UserTable(tables.Table):
+    name = tables.Column(verbose_name="Name")
```

Comment:
Is it intentional for the table to only have name? I see that this is intended to display users for everyone and not just admin, so it makes sense security wise. However, when signing up, you only need to use an email and password so it could just be empty for all users.

Reply:
<img width="576" alt="image" src="https://github.com/uchicago-cs/chigame/assets/111554249/9e62551d-2b5e-4afd-912b-0dbc3fe70bd0">
I think the name field is not optional but if it is, would having an email field help?

Reply:
It is optional. The name attribute lets blank=True. Email would work to make something show up. However, for the future functionalities that you guys want, this might not make sense as it will display emails to all users. What do you think? Maybe have a separate display for admin only with email and name for all users?

Reply:
Sorry missed the user-detail page, so I see you have one with the email!

Reply:
I've added the `first_name` and `last_name` attributes from the User model, but since `name` and these two attributes are all optional, if a User doesn't have any of these attributes, I'm not sure what we can do about it.

Reply:
Okay, this is fine with me!